MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -1287,7 +1286,7 @@ namespace WPEFramework {
                                 if (k_ret == 0){
                                     LOGINFO(" %s Termimated\n",script_names[i].c_str());
                                     /*this means we killed the task currently running */
-                                    task_incomplete = true;
+                                    m_task_map[task_names_foreground[i].c_str()]=false;
                                 }
```

Comment:
Shouldn't we break the loop here as we only run one task at a time and abort flag update should stop running any following scripts in line.